mono/mini/mini-posix.c

```diff
@@ -863,11 +863,13 @@ native_stack_with_gdb (pid_t crashed_pid, const char **argv, FILE *commands, cha
 	fprintf (commands, "attach %ld\n", (long) crashed_pid);
 	fprintf (commands, "info threads\n");
 	fprintf (commands, "thread apply all bt\n");
-	for (int i = 0; i < 32; ++i) {
-		fprintf (commands, "info registers\n");
-		fprintf (commands, "info frame\n");
-		fprintf (commands, "info locals\n");
-		fprintf (commands, "up\n");
+	if (!mini_get_debug_options ()->no_verbose_gdb) {
+		for (int i = 0; i < 32; ++i) {
```

Comment:
can we decrease the number of frames too? 10?

Reply:
It is too verbose, agreed. [author of verbosity]

Reply:
Might want to invert it.